### PR TITLE
Prepare release 2.55.0-rc.0

### PR DESCRIPTION
**Pull request by bboreham**
This is to prepare an RC for Prometheus v2.55.
The release-2.55 branch starts at commit ad652de29a3bbe2297e19f975de961388044ef26, just before the 3.0 branch was merged.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,39 @@
 
 ## unreleased
 
-* [FEATURE] OTLP receiver: Add new option `otlp.promote_resource_attributes`, for any OTel resource attributes that should be promoted to metric labels. #14200
-* [ENHANCEMENT] OTLP receiver: Warn when encountering exponential histograms with zero count and non-zero sum. #14706
+## 2.55.0-rc.0 / 2024-09-17
+
+* [CHANGE] TSDB: Only count unknown record types in `record_decode_failures_total` metric. #14042
```
Comment (bboreham) [inline CHANGELOG.md:7]: I could be swayed that this doesn't need to be announced as a CHANGE.

**Comment by bboreham** [comment]
I need to move the release-2.55 branch back to before the new UI was merged.

File: CHANGELOG.md (hunk 1 above)
Comment (jan--f) [reply CHANGELOG.md:7]: :+1:  it was originally proposed as a bug fix too: https://github.com/prometheus/prometheus/pull/14042/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR6

**Review by jan--f — APPROVED** [review]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (bboreham) [reply CHANGELOG.md:7]: I got it now.  That PR is both a bugfix and a misfeature.  I'll downgrade it from CHANGE.

**Pull request merged**